src/fights.ts

```diff
@@ -573,17 +574,25 @@ class FreeRunFight extends FreeFight {
   }
 }
 
-const pygmyMacro = Macro.if_(
-  $monster`pygmy bowler`,
-  Macro.trySkill($skill`Snokebomb`).item($item`Louder Than Bomb`)
-)
-  .if_(
-    $monster`pygmy orderlies`,
-    Macro.trySkill($skill`Feel Hatred`).item($item`divine champagne popper`)
-  )
-  .if_($monster`pygmy janitor`, Macro.item($item`tennis ball`))
-  .if_($monster`time-spinner prank`, Macro.basicCombat())
-  .abort();
+const pygmyMacro = () =>
+  Macro.if_($monster`pygmy bowler`, Macro.trySkill($skill`Snokebomb`).item($item`Louder Than Bomb`))
+    .if_(
+      $monster`pygmy orderlies`,
+      Macro.trySkill($skill`Feel Hatred`).item($item`divine champagne popper`)
+    )
+    .if_($monster`pygmy janitor`, Macro.item($item`tennis ball`))
+    .if_($monster`time-spinner prank`, Macro.basicCombat())
+    .externalIf(
+      SourceTerminal.have() || SongBoom.have(),
+      Macro.if_(
```

Comment:
do you actually get multiple rounds of stuff to use?

Reply:
No, but the intent is to only add one of them to the macro. In retrospect I can make this much, much simpler. 1 sec.